src/data/uts-isf/uts-isf_oecm_2024.json

```
@@ -0,0 +1,101 @@
+  "metric": [
+    "Emissions Intensity",
+    "Capacity",
+    "Generation",
+    "Technology Mix",
+    "Absolute Emissions"
+  ],
+  "keyFeatures": {
+    "emissionsTrajectory": "Significant decrease",
+    "energyEfficiency": "No information",
+    "energyDemand": "Low or no change",
+    "electrification": "Significant increase",
+    "policyTypes": ["Phaseout dates"],
+    "technologyCostTrend": "Decrease",
+    "technologyDeploymentTrend": "Signif. technology deployment",
```

```suggestion
    "technologyDeploymentTrend": "Minor technology deployment",
```
Another missed update here. Sorry